contrib/node/src/python/pants/contrib/node/tasks/node_install.py

```diff
@@ -0,0 +1,42 @@
+# coding=utf-8
+# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from pants.contrib.node.tasks.node_paths import NodePathsLocal
+from pants.contrib.node.tasks.node_task import NodeTask
+
+
+class NodeInstall(NodeTask):
+  """Installs a node_module target into the source directory
```

Comment:
It's not clear what the `source` directory is. This means "in the same directory as the target"?

Reply:
The first line of a task doc is the one that shows up in `./pants help $task`, so replacing the word "source" with something more meaningful would be good. Example:
```
$ ./pants help list

list options:
Lists all targets matching the target specs.
..
```